web/benchmark-core/src/jsMain/kotlin/com/sample/style/Stylesheet.kt

```diff
@@ -20,7 +20,7 @@ object AppCSSVariables : CSSVariables {
     val wtH3FontSize by variable<CSSUnitValue>()
     val wtH3LineHeight by variable<CSSUnitValue>()
 
-    val wtColCount by variable<Int>()
+    val wtColCount by variable<StylePropertyNumber>()
```

Comment:
Was it a mistake to use Int type there?

Reply:
Variable can have value which can have style property, now style property can be CSSStyleValue, StylePropertyString  (real String) and StylePropertyNumber  (real Number). Because we can't set it as Any, but we haven't CSSStyleValue|String|Number type in Kotlin. In real world there are not `Int` css value, there are units, even `number` unit fo simple numbers as `opacity`

Reply:
So it's a way to add constraints to CSS variable types, they can be only values which are allowed as style property value

Reply:
I will think about removing even StylePropertyNumber, to simplify type model